docs/monitoring/understanding-dashboards.md

```diff
@@ -145,14 +145,14 @@ The transaction processing panel tracks the time taken to complete the various t
 <Note>
 Block propagation was disabled in Geth at The Merge. Block propagation is now the responsibility of the consensus client.  Included here for archival interest.
 </Note>
- 
+
 Block propagation metrics track the rate that the local node hears about, receives and broadcasts blocks. This includes:
 
-- **ingress announcements**: the number of inbound announcements per second. Announcements are messages from peers that signal that they have a block to share.
+- **ingress announcements**: the number of inbound announcements per second. Announcements are messages from peers that signal that they have a block to share
 - **known announcements**: the number of announcements per second the local node is already aware of them
-- **malicious announcements**: the number of announcements per second that are determined to be malicious, e.g. because they are trying to mount a denial-of-service attack on the local node.
-- **ingress broadcasts**: the number of blocks directly propagated to local node per second.
-- **known broadcasts**: counts all blocks that have been broadcast by peers including those that are too far behind the head to be downloaded.
+- **malicious announcements**: the number of announcements per second that are determined to be malicious, e.g. because they are trying to mount a denial-of-service attack on the local node
+- **ingress broadcasts**: the number of blocks directly propagated to local node per second
+- **known broadcasts**: counts all blocks that have been broadcast by peers including those that are too far behind the head to be downloaded
 - **malicious broadcasts**: the number of blocks which are determined to be malicious per second
```

Comment:
IMO all of these are part-sentences (I don't know the english word for it), and should end with a `,`. Alternatively, they should end with `.`, but then they are separate sentences and should begin with a Literal. 

E.g: 

```
- **Ingress announcements**: The number of ... block to share. 
```
or
```
- **ingress announcements**: the number of ... block to share, 
```
Just my 5c, but I'm not a grammatician nor native english-speaker.